### ISSUE TITLE
about *.wav files errors prompts

### ISSUE THREAD

**Issue by lforlgg** [tigros/Whisperer#86]
about *.wav files errors prompts

if my audio files are many *.wav like:
1.wav
2.wav
3.wav
...

whisperer will pop up many windows prompts
---------------------------
Whisperer
---------------------------
Something went wrong, no output produced!

File name: D:\Temp\old\xx.wav.wav
---------------------------
OK 
---------------------------
It seems,enven if the audio file is *.wav,whisperer will also change the *.wav again to *.wav.wav?

Can it automatically skip the errors? 
When the final completion is completed, many errors are prompted at one time, rather than pop up many overlapping error prompt windows?

tks @tigros
💗

**Comment by tigros** [comment]
yes ur absolutely right my bad thanks for letting me know. just needed to rename .wav case before the success check.

replaced v3.6 just now.

thanks!
